docs/modules/ROOT/nav.adoc

```diff
@@ -1,27 +1,45 @@
+// Getting Started
 * xref:Getting Started.adoc[Getting Started]
```

Comment:
As I write below, the contents of getting started guide lading page is now useless. Let's not link this page and keep just the title